tests/conftest.py

```diff
@@ -23,16 +23,15 @@ def docker_compose(request):
 @pytest.fixture()
 def get_documents():
     def get_documents_inner(nr=10, index_start=0, emb_size=7):
-        random_batch = np.random.random([nr, emb_size]).astype(np.float32)
-        for i in range(index_start, nr + index_start):
+        for i in range(index_start, index_start + nr):
```

Comment:
Why did you change this? Please revert it, as it was optimized for the benchmarking tests